velox/functions/sparksql/Register.cpp

```diff
@@ -156,6 +157,8 @@ void registerFunctions(const std::string& prefix) {
   // Register size functions
   registerSize(prefix + "size");
 
+  registerArrayGetFunction(prefix + "get");
```

Comment:
We would suggest to register all the `array` functions to adjacent line.

https://github.com/facebookincubator/velox/blob/main/velox/functions/sparksql/Register.cpp#L304

Reply:
thank you, done.